.github/workflows/ci.yml

```diff
@@ -125,13 +125,13 @@ jobs:
           echo "Commit matrix: $(jq '.' commit-matrix.json)"
           echo "matrix=$(jq -c '.' commit-matrix.json)" >> $GITHUB_OUTPUT
 
-  check-commits-dispatcher:
+  check-commit:
     runs-on: ubuntu-latest
     needs: check-commits
```

Comment:
Don't we need to change to `check-commits-dispatcher`?  I guess the current CI is fake green.

Reply:
thank you, @ebyhr !

Reply:
I see it now in https://github.com/trinodb/trino/actions/runs/3998053503

```
[Invalid workflow file: .github/workflows/ci.yml#L130](https://github.com/trinodb/trino/actions/runs/3998053503/workflow)
The workflow is not valid. .github/workflows/ci.yml (Line: 130, Col: 12): Job 'check-commit' depends on unknown job 'check-commits'.
```

it's sad that it isn't getting propagated as a PR check result